lib/Controller/Schedule.php

```diff
@@ -1224,6 +1278,57 @@ public function edit($eventId)
             $this->getLog()->debug('Processed start is: FromDt=' . $fromDt->toRssString());
         }
 
+        // interrupt layout
+        if ($schedule->eventTypeId == 4) {
```

Comment:
Do we have a static variable for this ID? Might be better to use it here instead of `4`